python/ray/serve/_private/client.py

```diff
@@ -277,6 +277,7 @@ def deploy(
             version=version,
             route_prefix=route_prefix,
         )
+        controller_deploy_args.pop("ingress")
```

Comment:
Just some things about having to deal with v1 and v2 codepath ... `controller.deploy()` doesn't take the parameter ingress since it doesn't make sense in v1.

Reply:
oh, okay, this `deploy` func will be removed after v1 clean up, right?

Reply:
Yup!